pkg/catalogv2/helm/helm3.go

```diff
@@ -112,6 +121,7 @@ func fromHelm3ReleaseToRelease(release *release.Release, isNamespaced IsNamespac
 	return hr, err
 }
 
+// decodeHelm3 receives a helm3 release data and returns a pointer to the corresponding release.Release struct
```

Comment:
// decodeHelm3 receives a helm3 release data string, decodes the string data using base64 library
// and unmarshals the data into release.Release struct to return it.